src/components/dashboard/sidebar/OfferSummaryCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from 'react';
+
+import SidebarCard from './SidebarCard';
+import { UserSubsidyContext } from '../../enterprise-user-subsidy';
+
+export const OFFER_SUMMARY_TITLE = 'Assigned courses left to redeem';
+
+const OfferSideboardCard = () => {
```

Comment:
nit: should the component name match the file name? `OfferSideboardCard` --> `OfferSummaryCard`

Reply:
Yes, yes It should, but it's very out of scope for this PR.